### DISCUSSION TITLE
Install cloud custodian on kubernetes?

### DISCUSSION THREAD

**Discussion by caiopezzigft** [cloud-custodian/cloud-custodian#8838] [AWS] [answered]
Is it possible to install cloud custodian on kubernetes or only on docker, windows and linux?

**Comment by Eric-Fontana-Indico** [comment, marked as answer]
Yes, there is a helm chart, that is how I am using it.  See https://github.com/cloud-custodian/helm-charts/tree/main/charts/cloud-custodian-cron
